postgres/queries/organisations.sql

```diff
@@ -5,7 +5,8 @@ FROM
     organisations
 WHERE
     id = $1
-LIMIT 1;
+LIMIT
+    1;
```

Comment:
Is die LIMIT hier niet volledig overbodig? (Ik weet dat jij die niet hebt toegevoegd, maar ik zie het hier door die formatting)

Reply:
Ja inderdaad. Als we filteren op id kan dat er sowieso altijd maar 1 zijn

Reply:
Ik herinner mij vaag dat dat nodig was om sqlc duidelijk te maken dat hij maar 1 element moet teruggeven ipv een slice maar mogelijks leek dat zo omdat ik toen sqlc nog niet echt kende, die `:one` bovenaan zou al moeten zeggen dat er maar 1 element mag zijn 😛  je kan het alleszins wegdoen en eens kijken hoe de gegenereerde functie (al dan niet) verandert